pkg/kubernetes/diff.go

```diff
@@ -22,8 +22,16 @@ Please upgrade kubectl to at least version 1.18.1.`)
 
 	// required for separating
 	namespaces, err := k.ctl.Namespaces()
-	if err != nil {
-		return nil, errors.Wrap(err, "listing namespaces")
+	if err != nil || err == nil {
```

Comment:
```suggestion
	if err != nil {
```
right?

Reply:
Bah, that was a left-over debugging statement. I wanted to trigger the exception code even when an error didn't actually happen.